lib/tasks/copy_samples.rake

```diff
@@ -0,0 +1,167 @@
+require "aws-sdk-s3"
+
+DEFAULT_S3_REGION = "us-west-2".freeze
+BUCKET_NAME = "idseq-samples-#{Rails.env}".freeze
```

Comment:
use `SAMPLES_BUCKET_NAME` which we already have?

Reply:
Done